ios/Exponent.xcodeproj/xcshareddata/xcschemes/Exponent.xcscheme

```diff
@@ -90,6 +90,10 @@
       </EnvironmentVariables>
       <AdditionalOptions>
       </AdditionalOptions>
+      <LocationScenarioReference
+         identifier = "San Francisco, CA, USA"
+         referenceType = "1">
+      </LocationScenarioReference>
```

Comment:
🤔

Reply:
Without this option, asking for location on the simulator usually fails with `kCLErrorDomain` error as the location is unset by default. It can be fixed by choosing a fake location from simulator menus. I guess having a default location is what we want in most cases.